DISTMYSQL-94 Added README, License and Contributing guides (#17)
percona/pdmysql-docs: CONTRIBUTING.md

# Contributing Guide

Thank you for deciding to contribute and help us improve Percona Distribution for MySQL documentation!

> We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).

Reviewer: I suggest to use the rendered online version instead: https://percona.community/contribute/coc/

Author: Fixed

Reviewer: Somehow that change does not seem to have made it into your last push?

Author: Strange. Now it must look OK

5. Someone from our team reviews the pull request and if everything is correct, merges it into the documentation. Then it gets published on the site.

### Edit documentation locally

This option is for users who prefer to work from their computer and / or have the full control over the documentation process.

The steps are the following:

1. Fork this repository
2. Clone the repository on your machine:

```sh
git clone git@github.com:<your_name>/pdmysql-docs.git
```

3. Change the directory to ``pdmysql-docs`` and add the remote upstream repository:

```sh
git remote add upstream git@github.com:percona/pdmysql-docs.git
```

4. Create a separate branch for your changes

```sh
git checkout -b <my_changes>
```

5. Make changes
6. Commit your changes
7. Open a pull request to Percona

### Building the documentation

> To verify how your changes look, generate the static site with the documentation. This process is called *building*. You can do it in these ways:

Reviewer: ```suggestion
To verify the layout of your changes, generate the static site with the documentation. This process is called *building*. You can do it in these ways:
```
